cirq-web/README.rst

```diff
@@ -48,15 +92,18 @@ with the following:
     sphere = BlochSphere(state_vector=state_vector)
     display(sphere)
 
-See the example Jupyter notebook in this directory for more examples on how to use cirq-web.
+You can find more example Jupyter notebooks in the |cirq-web| subdirectory of
+the |cirq-github|_.
+
+
```

Comment:
maybe remove these blanks?

Reply:
I was using 2 blank lines before section headings, just as a convention. It's not uncommon if you all prefer single blank lines, that's okay with me.

Reply:
Turns out the Google Markdown convention does say 1 blank line, so in the Markdown version of the files, there is 1 blank line before and after section headings.